### PR TITLE
fix: remove `ropsten`, `kovan`, and `rinkeby`.

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What I did

Ropsten, Kovan, and Rinkeby are all done. This PR removes their existence.
Hello Goerli, my new best friend!

Also changes the required confs in mainnet to be `2` instead of `13`, because of PoS.

Also no longer added PoA middleware to Goerli since it is now PoS

### How I did it

A big flamethrower and I burnt it all.
What could be salvaged was handed to Goerli.

### How to verify it

It doesn't seem like this breaks any of the plugins, which is what I was curious about.

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by fubuloubu — APPROVED** [review]

File: src/ape_geth/provider.py [hunk 1]
```diff
@@ -251,8 +248,8 @@ def is_likely_poa() -> bool:
                 else "Error getting chain id."
             )
 
-        # If network is rinkeby, goerli, or kovan (PoA test-nets)
-        if chain_id in (4, 5, 42) or is_likely_poa():
+        # If network is goerli or a custom PoA testnet
```
Comment (unparalleled-js) [inline src/ape_geth/provider.py:251]: Wait - Goerli is Proof of Stake now, so this should not be needed, right? @fubuloubu

**Review by NotPeopling2day — APPROVED** [review]
🔥

File: src/ape_geth/provider.py (hunk 1 above)
Comment (fubuloubu) [reply src/ape_geth/provider.py]: Yes! This should be true

**Review by fubuloubu — APPROVED** [review]

**Review by NotPeopling2day — APPROVED** [review]

**Comment by unparalleled-js** [comment]
Related PRs:

https://github.com/ApeWorX/ape-arbitrum/pull/5
https://github.com/ApeWorX/ape-polygon/pull/8
https://github.com/ApeWorX/ape-fantom/pull/16
https://github.com/ApeWorX/ape-optimism/pull/9
https://github.com/ApeWorX/ape-etherscan/pull/49
https://github.com/ApeWorX/ape-alchemy/pull/31
https://github.com/ApeWorX/ape-infura/pull/52

**Pull request merged**